components/Nav/Burger.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import RightNav from './RightNav';
+
+/*This code is divided into 3 parts, the Styled burgerbar formed when the screen become mobile size, when clicked its Open state becomes tru and is paased on RightBar to change the navigation view */
+const StyledBurger = styled.div`
+width: 2rem;
+height: 2rem;
+position: fixed;
+top: 15px;
+right: 20px;
+z-index: 500;
+display: none;
+  @media (max-width: 768px) {
```

Comment:
- There's a few responsiveness issues

Reply:
ohk, will increasing the max-width help?

Reply:
Yea, it should